charts/langsmith/values.yaml

```diff
@@ -219,6 +219,9 @@ frontend:
   name: "frontend"
   containerPort: 8080
   existingConfigMapName: ""
+  # Turn on LLM Proxy Cache
+  cache:
```

Comment:
should this be on or off?

Reply:
Probably default on? People need to explicitly change oai url to use anyways?

Reply:
also any configuration params we should add here? I can also add some otehr ones like anthropic